### DISCUSSION TITLE
suggest horizontal scrolling bar on the top of the tables

### DISCUSSION THREAD

**Discussion by Smailancer** [filamentphp/filament#12484] [General]
Hello there , I want to point to a user experience little problem , if want to see the whole first long entry in a table and the table is full with data  we must scroll down to the horizontal scroller , than scroll to the right of the table than scroll up back to the top of the table to see the rest of the first line , so my suggestion is to add another horizontal scroller on the top of the table to make the user experience of tables better . 
![image](https://github.com/filamentphp/filament/assets/15679510/916f785b-f3e0-4d55-8672-cc1cbabd8464)

**Comment by dissto** [comment]
You can hold `Shift` while scrolling to scroll horizontally. That's pretty common for tables on the web. And ofc Filament supports it.

Alternatively you could [`wrap`](https://filamentphp.com/docs/3.x/tables/columns/text#wrapping-content) the columns content or [`limit`](https://filamentphp.com/docs/3.x/tables/columns/text#limiting-text-length) it.

**Reply by Merdin** [reply]
What if you have a customer that specifically requests this, and we are able to add the functionality, why not add support for it?
